lib/sidekiq/extensions/action_mailer.rb

```diff
@@ -27,19 +28,20 @@ def perform(yml)
     end
 
     module ActionMailer
-      def sidekiq_delay(options={})
+      def sidekiq_delay(options = {})
         Proxy.new(DelayedMailer, self, options)
       end
-      def sidekiq_delay_for(interval, options={})
-        Proxy.new(DelayedMailer, self, options.merge('at' => Time.now.to_f + interval.to_f))
+
+      def sidekiq_delay_for(interval, options = {})
+        Proxy.new(DelayedMailer, self, options.merge("at" => Time.now.to_f + interval.to_f))
       end
-      def sidekiq_delay_until(timestamp, options={})
-        Proxy.new(DelayedMailer, self, options.merge('at' => timestamp.to_f))
+
+      def sidekiq_delay_until(timestamp, options = {})
+        Proxy.new(DelayedMailer, self, options.merge("at" => timestamp.to_f))
       end
-      alias_method :delay, :sidekiq_delay
-      alias_method :delay_for, :sidekiq_delay_for
-      alias_method :delay_until, :sidekiq_delay_until
+      alias delay sidekiq_delay
```

Comment:
⚠️ I not sure about this change. In real this not break stuff but what's about code base using monkey patch to override alias? (I plan to test this to ensure no regression for the end user)

Reply:
alias_method operates more like a normal Ruby method so it keeps with the POLA. I try to avoid "weird" Rubyisms like alias.  https://en.wikipedia.org/wiki/Principle_of_least_astonishment